2.3.c

```diff
@@ -0,0 +1,43 @@
+#include <stdio.h>
+
+//�������� ���������� ��������� � ��������.
+
+int bubbleSort(int* num, int size)
+{
+	for (int i = 0; i < size - 1; i++)
+	{
+		for (int j = (size - 1); j > i; j--)
+		{
+			if (num[j - 1] > num[j])
+			{
+			int s = num[j - 1];
+			num[j - 1] = num[j];
+			num[j] = s;
+			return 0;
+			}
+	
+		}
+	}
+
+}
+ 
+
+int main()
+{
+	int a[10];

```

Comment:
```suggestion
	int a[ARRAY_SIZE] = { 0 };
```
Переменные нужно инициализировать.
Размер массива вынеси в define.
И 10 - очень маленький размер.
Либо выдели с запасом, а пользователь пусть вводит размер массива и элементы.
Либо узнай у пользователя размер массива и выдели массив с помощью `calloc`.